HealthGPT/HealthGPTAppDelegate.swift

```diff
@@ -30,24 +30,19 @@ class HealthGPTAppDelegate: SpeziAppDelegate {
             }
             HealthDataInterpreter()
             HealthDataFetcher()
-            SecureStorage()
+            KeychainStorage()
         }
     }
 
 
     private var healthKit: HealthKit {
         HealthKit {
```

Comment:
We should probably aim to replace the existing manual queries with Spezi HealthKit wrappers:
https://swiftpackageindex.com/StanfordSpezi/SpeziHealthKit/1.1.3/documentation/spezihealthkit

Reply:
Not really convinced this would actually make the code here simpler, but probably a good idea to use everything the package provides since this is meant to be an example project.

Reply:
Great idea! Will address this in the next PR.